client/js/verification-portal/resource-view.js

```diff
@@ -3,7 +3,7 @@ import injectSheet from 'react-jss'
 
 import HeaderModal from './components/header-modal'
 import Spacer from '../components/spacer'
-import LoginButton from '../authentication/facebook-login-button'
+import FacebookLoginButton from '../authentication/facebook-login-button'
```

Comment:
@vzajkov The linter is failing. I think it's because this is not actually used and can be removed.